protocol-designer/src/pages/Designer/DeckSetup/LabwareTools.tsx

```diff
@@ -283,7 +283,11 @@ export function LabwareTools(props: LabwareToolsProps): JSX.Element {
                             buttonValue={labwareURI}
                             onChange={e => {
                               e.stopPropagation()
-                              setSelectedLabwareDefURI(labwareURI)
+                              setSelectedLabwareDefURI(
+                                labwareURI === selecteLabwareDefURI
+                                  ? null
+                                  : labwareURI
+                              )
```

Comment:
maybe the following would be better 🤔 

```suggestion
                              setSelectedLabwareDefURI((prevSelectedLabwareDefURI) => (prevSelectedLabwareDefURI === labwareURI ? null : labwareURI));
```